ParabolInc/action
# add download csv to email fix #2453

TEST (how to do it locally)
- run through a retro meeting through completion, save the meetingId
- add the meetingId to line 18 of 'emailSSR.js'
- [ ] go to localhost:3000/email, see an email summary with an export to csv button
- [ ] click the export to csv button
- [ ] a new window opens with the meeting summary & the export will download
